CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.1] - 2018-08-17
+### Added
+- Catch exception when tube not found for ``beanstalkd/api/tubes/{tube}`` and ``beanstalkd/tubes/{tube}``.
```

Comment:
Please don't update the changelog in this changeset. I will update it before releasing a new version. The release date as well as the added/changed features might still change.